website/client/src/components/settings/site.vue

```diff
@@ -739,6 +739,14 @@ export default {
       } else if (attribute === 'email') {
         this.user.auth.local.email = updates.newEmail;
         window.alert(this.$t('emailSuccess')); // eslint-disable-line no-alert
+      } else if (attribute === 'password') {
+        this.passwordUpdates = {};
+        this.$store.dispatch('snackbars:add', {
+          title: 'Habitica',
+          text: 'Your password has been changed',
```

Comment:
This string ought to be translatable. Let's add this to `website/common/locales/en/settings.json` and call `this.$t` to supply the confirmation in the user's preferred language if available.

Reply:
I committed the change using the internationalization. There is already the 'passwordChangeSuccess' but I still added the 'passwordSuccess' to website/common/locales/ since the 'passwordChangeSuccess'  seems to be too long.